tensorly/decomposition/_cp.py

```diff
@@ -446,6 +446,9 @@ def sample_khatri_rao(matrices, n_samples, skip_matrix=None,
     skip_matrix : None or int, optional, default is None
         if not None, index of a matrix to skip
 
+    indices_list : list, default is None
+        if None, random indices will be created
```

Comment:
Can we add a longer description, e.g. what is selected and range of the indices?

Reply:
I added some information, I hope it is ok for you